en/deployment/articles/03-installing-a-search-engine/02-elasticsearch/07-security.markdown

```diff
@@ -169,27 +175,47 @@ configuration (`elasticsearch.yml`) used in this guide (with the Elasticsearch 6
 example commented out):
 
 ```yaml
-# For Elasticsearch 7
+# For Elasticsearch 7.3/7.4
 cluster.name: LiferayElasticsearchCluster
+
+# X-Pack Security
 xpack.security.enabled: true
+
+## TLS/SSL settings for Transport layer
 xpack.security.transport.ssl.enabled: true
-xpack.security.transport.ssl.verification_mode: certificate
-xpack.security.transport.ssl.key: /path/to/[Elasticsearch Home]/config/localhost.crt
-xpack.security.transport.ssl.certificate: /path/to/[Elasticsearch Home]/config/localhost.key
-xpack.security.transport.ssl.certificate_authorities: ["/path/to/ca.crt"]
+xpack.security.transport.ssl.verification_mode: certificate 
+xpack.security.transport.ssl.key: /path/to/localhost.key
+xpack.security.transport.ssl.certificate: /path/to/localhost.crt
+xpack.security.transport.ssl.certificate_authorities : ["/path/to/ca.crt"]
+
+# TLS/SSL settings for HTTP layer
 xpack.security.http.ssl.enabled: true
-xpack.security.http.ssl.certificate: /path/to/[Elasticsearch Home]/config/localhost.crt
-xpack.security.http.ssl.key: /path/to/[Elasticsearch Home]/config/localhost.key
-xpack.security.http.ssl.certificate_authorities: ["/path/to/ca.crt"]
+xpack.security.http.ssl.verification_mode: certificate 
+xpack.security.http.ssl.key: /path/to/localhost.key
+xpack.security.http.ssl.certificate: /path/to/localhost.crt
+xpack.security.http.ssl.certificate_authorities : ["/path/to/ca.crt"]
+
+# Comment out when Kibana and Liferay's X-Pack Monitoring are also configured
+#xpack.monitoring.collection.enabled: true
```

Comment:
Why include the monitoring setting in the security documentation?

Reply:
It's commented out, but added just for reference. Because people will start with configuring security I thought it's worth to add.

But feel free to remove.